src/condor/implementations/iterative.py

```diff
@@ -152,7 +152,7 @@ def __call__(self, model_instance):
         for k, v in model_instance.variable.asdict().items():
             model_var = getattr(self.model, k)
             if (
-                model_var.warm_start
+                np.array(model_var.warm_start).any()
```

Comment:
Does `all(model_var.warm_start)` work? It should work with any iterable of bool-ish elements and would save a copy/alloc.

Reply:
Thanks, I didn't know that one